pkg/operator/ceph/object/objectstore.go

```diff
@@ -1000,3 +999,38 @@ func errorOrIsNotFound(err error, msg string, args ...string) error {
 	}
 	return errors.Wrapf(err, msg, args)
 }
+
+// check whether zone endpoint list need to be updated or not
+func UpdateZoneEndpointList(zones []zoneType, desiredEndpointList []string, zoneName string) bool {
+	updateZoneEndpointList := false
+	isZonePresent := false
+	zoneEndpointsList := []string{}
+	checkZoneEndpoints := make(map[string]bool)
+	if zoneName != "" {
```

Comment:
If we are going to handle the case of a zone name being passed as an empty string it should return an error about the invalid input. We should not silently ignore usage errors.